CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 - Command Generate - Generate the forest routes to ease the use of `smart routes`.
 
 ### Changed
+- Command Generate - Use handlebars to generate models. 
```

Comment:
This was a technical change

Reply:
It's missing a fix, you fixed the targetKey unconventionnal when in undersocred is false